io.github.riftr.intelpy.yml

```diff
@@ -0,0 +1,65 @@
+app-id: io.github.riftr.intelpy
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+name: intelpy
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --filesystem=~
+  - --socket=pulseaudio
+build-options:
+  env:
+    QT_DEBUG_PLUGINS: '1'
+
```

Comment:
You need to cleanup the baseapp from development files https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp?tab=readme-ov-file#example-pyqtwebengine-application cleanup-commands.